content/400-reference/200-api-reference/100-prisma-schema-reference.mdx

```diff
@@ -2319,6 +2355,18 @@ The name of the `name` argument on the `@relation` attribute can be omitted (`re
 @relation(_ name: String?, fields: FieldReference[]?, references: FieldReference[]?)
 ```
 
+Starting with 2.28 if the `referentialActions` preview flag is enabled the signature changes.
```

Comment:
```suggestion
As of 2.28.0, if the `referentialActions` preview flag is enabled the signature changes.
```

Reply:
Come to think of it, ref actions is going GA with v3. But i guess we can remove this notice then

Reply:
Same is true for named Constraints. We can remove / change all these notices then